packages/charts/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -760,9 +774,10 @@ export class VerticalStackedBarChartBase
         item.shouldDrawBorderBottom = true;
       });
     }
+
     this.setState({
       refSelected,
-      isCalloutVisible: true,
+      isCalloutVisible: stack.chartData.length > 0 || (stack.lineData?.length ?? 0) > 0,
```

Comment:
>stack.chartData.length > 0 || (stack.lineData?.length ?? 0) > 0, [](http://example.com/codeflow?start=24&length=64)

is this condition possible

Reply:
yes, for stacks which does not have line data, checked it explicitly. For example, in the second data point, there is no line data.
`[
{
        x: 80000,
        y: 20000,
        legend: 'Elephants',
        color: getColorFromToken(DataVizPalette.color11),
        xAxisCalloutData: '2020/04/30',
        yAxisCalloutData: '8%',
        lineData: {
          y: 16000,
          yAxisCalloutData: '7%',
        },
      },
      {
        x: 92000,
        y: 45000,
        legend: 'Monkeys',
        color: getColorFromToken(DataVizPalette.color6),
        xAxisCalloutData: '2020/04/30',
        yAxisCalloutData: '19%'
]`

Without this check, it will show something like:
![image](https://github.com/user-attachments/assets/900919db-ac7b-4bfd-af17-7bfa39c05e72)